integration-tests/src/test/resources/observability/tracing_tests/Config.toml

```
@@ -0,0 +1,3 @@
+[ballerina.observe]
+tracingEnabled=true    # Flag to enable Metrics
```

Let's fix this mistake in the comment (not part of this migration though)
```suggestion
tracingEnabled=true    # Flag to enable Tracing
```

---

Fixed with https://github.com/ballerina-platform/module-ballerinai-observe/pull/1/commits/4b2218280291e2dee542697481cd43e7f8469747